### PR TITLE
chore: add changeLog to storybook

### PR DESCRIPTION
**Pull request by JimmFly**
work for #1998

### CODE REVIEW

File: packages/component/src/components/changeLog/index.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  };
+  return (
+    <div
+      className={
+        isClose ? changeLogWrapperSlideOutStyle : changeLogWrapperSlideInStyle
+      }
+    >
+      <div
+        data-testid="change-log"
+        className={isClose ? changeLogSlideOutStyle : changeLogSlideInStyle}
```
Comment (himself65) [inline packages/component/src/components/changeLog/index.tsx]: combine with `clsx`

**Review by Himself65 — CHANGES_REQUESTED** [review]

File: packages/component/src/components/changeLog/index.tsx (hunk 1 above)
Comment (himself65) [reply packages/component/src/components/changeLog/index.tsx]:
```ts
className={clsx({
  changeLogSlideOutStyle: isClose,
  changeLogSlideInStyle: !isClose
})}
```

Comment (himself65) [reply packages/component/src/components/changeLog/index.tsx]: see https://www.npmjs.com/package/clsx

Comment (JimmFly) [reply packages/component/src/components/changeLog/index.tsx:35]: got it

**Review by Himself65 — APPROVED** [review]

**Pull request merged**